test/100-bridge-iptables.bats

```diff
@@ -150,7 +150,7 @@ fw_driver=iptables
     run_helper ps "$aardvark_pid"
     assert "${lines[1]}" =~ ".*aardvark-dns --config $NETAVARK_TMPDIR/config/aardvark-dns -p $dns_port run" "aardvark not running or bad options"
 
-    NETAVARK_DNS_PORT="$dns_port" run_netavark --file ${TESTSDIR}/testfiles/dualstack-bridge-network-container-dns-server.json \
+    NETAVARK_DNS_PORT="$dns_port" run_helper --file ${TESTSDIR}/testfiles/dualstack-bridge-network-container-dns-server.json \
         update podman1 --network-dns-servers 8.8.8.8
```

Comment:
I think you need to add `assert "$output" = ""` or something similar. Remember, the message does not cause a program failure or error exit status.